pennylane/compiler/python_compiler/universe.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2025 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""xDSL universe for containing all dialects and passes."""
+
+xdsl_available = True
+
+try:
+    from xdsl.passes import ModulePass
+    from xdsl.universe import Universe
+except (ImportError, ModuleNotFoundError):
```

Comment:
```suggestion
except ModuleNotFoundError:
```
Can we just keep this `ModuleNotFoundError`?